### PR TITLE
Run 2 threads for release raft builds

### PR DESCRIPTION
**Pull request by ashamis**

### CODE REVIEW

File: cmake/common.cmake [hunk 1]
```diff
@@ -373,7 +373,12 @@ else()
   set(CONSENSUS_ARG "raft")
 endif()
 
-set(WORKER_THREADS 0)
+if("Release" IN_LIST CMAKE_BUILD_TYPE AND NOT PBFT)
```
Comment (achamayou) [inline cmake/common.cmake]: This won't work I believe, our release builds are actually RelWithDbgInfo or something like that.

Comment (olgavrou) [reply cmake/common.cmake]: `RelWithDebInfo`

Comment (eddyashton) [reply cmake/common.cmake]: `CMAKE_BUILD_TYPE` should be a single var, not a list, and I think this should actually be a `NOT Debug` check (ie - we want 0 worker threads in `Debug`, but 2 for everything else).

https://cmake.org/cmake/help/latest/variable/CMAKE_BUILD_TYPE.html

**Comment by cimetrics** [comment]
threading@[4766 aka 20200211.4](https://msrc-ccf.visualstudio.com/CCF/_build/results?buildId=4766&view=results) vs master ewma over 30 builds from [4442](https://msrc-ccf.visualstudio.com/CCF/_build/results?buildId=4442&view=results) to [4762](https://msrc-ccf.visualstudio.com/CCF/_build/results?buildId=4762&view=results)
images

**Comment by achamayou** [comment]
We are getting very bad performance with this on the distributed setup:

```
46: 2020-02-11 12:04:33.549 | DEBUG    | infra.remote:_print_upload_perf:101 - 200003 transactions took 13039ms.
46: 2020-02-11 12:04:33.549 | DEBUG    | infra.remote:_print_upload_perf:101 - 	=> 15338.8tx/s
46: 2020-02-11 12:04:33.549 | SUCCESS  | infra.remote:_print_upload_perf:105 - METRICS: SB_SGX_CFT^ = 15338.8
46: 2020-02-11 12:04:33.549 | DEBUG    | infra.remote:_print_upload_perf:101 - Sends: 200003
```

**Comment by achamayou** [comment]
Ah, but that looks like it's caused by adding the second perf instance, without merging the PR that controls the dispatch, so both CFT and BFT tests ran around the same time on the same kit. Let's have another look once #823 is in.

**Comment by achamayou** [comment]
Numbers look somewhat better (although not as much as expected) now that dispatch is fixed, merging.

**Pull request merged**